docs/tutorial/index.rst

```diff
@@ -0,0 +1,270 @@
+On the authorization page, click the green "Authorize readthedocs" button.
+
+.. figure:: /_static/images/tutorial/github-authorization.png
+   :width: 60%
+   :align: center
+   :alt: GitHub authorization page
+
+   GitHub authorization page
+
+.. note::
+
+   Read the Docs needs elevated permissions to perform certain operations
+   that ensure that the workflow is as smooth as possible,
+   like installing webhooks.
+   If you want to learn more,
+   check out :ref:`connected-accounts:permissions for connected accounts`.
+
+After that, you will be redirected to Read the Docs,
+where you will need to confirm your e-mail and username.
+Clicking the "Sign Up »" button will create your account
+and redirect you to your *dashboard*.
+
+By now, you should have two email notifications:
+
+* One from GitHub, telling you that "A third-party OAuth application ...
+  was recently authorized to access your account". You don't need to do
+  anything about it.
+* Another one from Read the Docs, prompting you to "verify your email
+  address". Click on the link to finalize the process.
+
+Finally, you created your account on Read the Docs
+and are ready to import your first project.
+
+Welcome!
+
+.. figure:: /_static/images/tutorial/rtd-empty-dashboard.png
+   :width: 80%
+   :align: center
+   :alt: Read the Docs empty dashboard
+
+   Read the Docs empty dashboard
+
+.. note::
+
+   Our commercial site offers some extra features,
+   like support for private projects.
+   You can learn more about :doc:`our two different sites </choosing-a-site>`.
+
+First steps
+-----------
+
+Importing the project to Read the Docs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To import your GitHub project to Read the Docs,
+first click on the "Import a Project" button on your dashboard
+(or browse to `the import page <https://readthedocs.org/dashboard/import/>`_ directly).
+You should see your GitHub account under the "Filter repositories" list on the right.
+If the list of repositories is empty, click the "refreshing your accounts" link,
+and after that all your repositories will appear on the center.
+
+.. figure:: /_static/images/tutorial/rtd-import-projects.png
+   :width: 80%
+   :align: center
+   :alt: Import projects view
+
+   Import projects view
+
+Locate your ``rtd-tutorial`` project
+(possibly clicking "next ››" at the bottom if you have several pages of projects),
+and then click on the "+" button to the right of the name.
+The next page will ask you to fill some details about your Read the Docs project:
+
+Name
+  The name of the project. It has to be unique across all the service,
+  so it is better if you prepend your username,
+  for example ``astrojuanlu-rtd-tutorial``.
+
+Repository URL
+  The URL that contains the sources. Leave the automatically filled value.
+
+Repository type
+  Version control system used, leave it to "Git" (which should be the default).
+
+Default branch
+  Name of the default branch of the project, leave it to ``main``.
```

Comment:
Are we sure it will be `main`? Is this configured in our template or on GH somewhere? 

```suggestion
  Name of the default branch of the project, leave it as ``main``.
```

Reply:
Yep, it's `main` in the repository itself, and so when the user "uses the template", the branch will also be called `main`. So happy that I don't need to juggle two possible branch names here 😄